src/lib/mongoBackend/mongoRegistrationGet.cpp

```diff
@@ -216,20 +216,9 @@ static void setExpires(ngsiv2::Registration* regP, const mongo::BSONObj& r)
 *
 * setStatus -
 */
-static void setStatus(ngsiv2::Registration* regP, const mongo::BSONObj& r)
-{
-  regP->status = (r.hasField(REG_STATUS))? getStringFieldF(r, REG_STATUS): "";
-}
-
-
-
-/* ****************************************************************************
-*
-* setForwardingInformation -
-*/
-static void setForwardingInformation(ngsiv2::Registration* regP, const mongo::BSONObj& r)
+static void setStatus(ngsiv2::Registration* regP, const mongo::BSONObj& r, const char* dbLabel)
```

Comment:
Function is overparametriced, `dbLabel` is not needed at the field used for status is not a parameter but fixed by the database model.

Reply:
Fixed in 5d851ef